src/Listener/ToolbarListener.php

```diff
@@ -135,7 +135,8 @@ protected function injectToolbar(ProfilerEvent $event)
             $response->getBody(),
             1
         );
-        $injected = preg_replace('/<\/head>/i', $style . "\n</head>", $injected, 1);
+        $prepend = preg_match('/<\/head>/i', $injected) ? 'head' : 'body';
+        $injected = preg_replace('/<\/' . $prepend . '>/i', $style . "\n</$prepend>", $injected, 1);
```

Comment:
And what about the case when no `body` element exists?

Reply:
do you have suggestion?

Reply:
Check always for `head` and `body`. If both does not present, add the HTML of the toolbar to the end.

Reply:
implemented, please verify